controllers/memberoperatorconfig/cache.go

```diff
@@ -45,27 +49,50 @@ func loadLatest(cl client.Client, namespace string) error {
 		}
 		return err
 	}
-	configCache.set(config)
+
+	allSecrets, err := loadSecrets(cl)
+	if err != nil {
+		return err
+	}
+
+	configCache.set(config, allSecrets)
 	return nil
 }
 
+func loadSecrets(cl client.Client) (map[string]map[string]string, error) {
+	var allSecrets = make(map[string]map[string]string)
+	secretList := &corev1.SecretList{}
+	err := cl.List(context.TODO(), secretList)
```

Comment:
Shouldn't we list in specific namespace?

Reply:
Yes, thanks for pointing it out. Done in c74fe7da69590d0d2861b41845039aaf0c19fffb